sdk/testing/stepwise/stepwise.go

```diff
@@ -0,0 +1,351 @@
+// Package stepwise offers types and functions to enable black-box style tests
+// that are executed in defined set of steps. Stepwise utilizes "Environments" which
+// setup a running instance of Vault and provide a valid API client to execute
+// user defined steps against.
+package stepwise
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	log "github.com/hashicorp/go-hclog"
+
+	"github.com/hashicorp/vault/api"
+	"github.com/hashicorp/vault/sdk/helper/consts"
+	"github.com/hashicorp/vault/sdk/helper/logging"
+)
+
+// StepOperation defines operations each step could preform
+type StepOperation string
```

Comment:
This was intentional; I wanted developers to consistently use the `stepwise` package when building these tests, instead of using mostly `stepwise` and then 1 part (the operations) from the `logical` package. 

I've updated the code comment in c88cff0fb to reflect this.